### ISSUE TITLE
Spec is sometimes string, sometimes array

### ISSUE THREAD

**Issue by SachaG** [web-platform-dx/web-features#1981]
It seems like the `spec` field can either be a string or an array? Maybe that's on purpose, but I just wanted to point out that this can be a little inconvenient for typed APIs such as GraphQL. Could it be standardized to always be an array?

**Comment by captainbrosset** [comment]
I agree that this is inconvenient. In the explorer, I've had to add some code to always make it an array: https://github.com/web-platform-dx/web-features-explorer/blob/main/.eleventy.js#L59-L64

**Comment by SachaG** [comment]
Yeah that's also what I did for now.

**Comment by ddbeck** [comment]
Two consumer complaints. To me, that sounds like justification enough to make it always an array.

For background, I think we did it mostly as a convenience to authors. Now that we have a build step anyway, there's no reason we can't make the type consistently `string[]` at build time.

I think, with the next breaking version, we ought to do this for all the `string | [string, string, ...string[]]` fields:

- `spec`
- `group`
- `snapshot`
- `caniuse`

**Comment by SachaG** [comment]
I think `group` has the same issue?

**Comment by jgraham** [comment]
FWIW I also ran into this and had to write a similar string-or-array-to-array helper, so I'm also in favour of this change. _Arguably_ it's non-breaking (I suppose someone might be relying on arrays always being length > 1 but it seems at least a little bit unlikley).
